### ISSUE TITLE
Remove CSS settings from app-settings

### ISSUE THREAD

**Issue by ns8482e** [OrchardCMS/OrchardCore#17235]
Today the CSS classes are defined in app-settings as following, this makes app-settings noisy - The purpose of app setting to be limited to application and feature configuration - that managed by configuration management and not for UX/UI management

```json
   //"TheAdminTheme": {
    //  "StyleSettings": {
    //    "WrapperClasses": "mb-3",
    //    "LimitedWidthWrapperClasses": "row",
    //    "LimitedWidthClasses": "col-md-6 col-lg-4 col-xxl-3",
    //    "StartClasses": "",
    //    "EndClasses": "",
    //    "LabelClasses": "",
    //    "OffsetClasses": ""
    //   }
    //},
```
Like Zone, we don't provide Zones configuration in app-settings, instead we provide UI to manage zones 
Similarly we should not have CSS settings in app settings -instead, theme should provide custom UI to manage such settings needed for the theme that is only applicable to that theme.

Possible options

- Use HTML to define CSS  classes
- Use custom tag helper to inject custom classes instead of C# Helper methods
- Provide Settings UI for the theme if specific theme provides configurable items for UI/UX users.

**Comment by MikeAlhayek** [comment]
These options are used by the these helpers that are used everywhere 

https://github.com/OrchardCMS/OrchardCore/blob/411cbf1a60d970722395eef827f333c18977e18a/src/OrchardCore/OrchardCore.ContentManagement.Display/Razor/CssOrchardHelperExtensions.cs#L8

https://github.com/OrchardCMS/OrchardCore/blob/411cbf1a60d970722395eef827f333c18977e18a/src/OrchardCore/OrchardCore.DisplayManagement/Html/CssOrchardHelperExtensions.cs#L8


These configuration can be configured by any theme that is derived by TheAdmin theme to enable styling and small customization. Until we reusable components in TheAdmin theme that allows one to override the look and feel of TheAdmin theme, these should be be removed.

For example, I have an admin theme that places the labels on the same line as the input. I am able to do this now using these options. Here is an example

![image](https://github.com/user-attachments/assets/00af19d3-a71b-4b7c-a59f-4ab1ec408efa)

This is possible by adding the following 

```
        services.PostConfigure<TheAdminThemeOptions>(options =>
        {
            options.WrapperClasses = "row mb-3";
            options.LimitedWidthWrapperClasses = "row";
            options.LimitedWidthClasses = "col-md-6 col-lg-5 col-xxl-4";
            options.StartClasses = "col-lg-2 col-xl-3";
            options.EndClasses = "col-lg-10 col-xl-9";
            options.LabelClasses = "col-form-label text-lg-end col-lg-2 col-xl-3";
            options.OffsetClasses = "offset-lg-2 offset-xl-3";
        });
```

**Comment by hishamco** [comment]
From a UI/UX perspective coding CSS classes is hard, it would be nice to let the HTML control everything. I still remember one of my friends 3 or 4 years ago when he told me, why the framework is tightly coupled on Bootstrap, this will limit the possibility of extending or changing the UI framework in the future

**Comment by MikeAlhayek** [comment]
> why the framework is tightly coupled on Bootstrap, this will limit the possibility of extending or changing the UI framework in the future

I agree. That's when having some sort of components design would be great. Until then, these options should stay to allow customizing our tightly couple theme.

**Comment by hishamco** [comment]
I created a prototype long time ago, hopefully one day I will demo it
